functions/Copy-DbaAgentJob.ps1

```diff
@@ -248,6 +248,8 @@ function Copy-DbaAgentJob {
                             $sql = $sql -replace [Regex]::Escape("@owner_login_name=N'$missingLogin'"), [Regex]::Escape("@owner_login_name=N'$saLogin'")
                         }
 
+                        $sql = $sql -replace [Regex]::Escape("@server=N'$($sourceserver.DomainInstanceName)'"), [Regex]::Escape("@server=N'$($destServer.DomainInstanceName)'")
```

Comment:
FYI #1383 was why we removed this in most every copy command.

Reply:
Thank you. I recalled that convo but vaguely so I was happy to revisit it. 

The replacement ck talked about was too broad. This one is specifically for the impacted server ala @server=. Without it, I couldn't even migrate a number of my jobs. I had to export the script then modify manually.